src/js/core/range.js

```diff
@@ -605,7 +605,25 @@ define([
        * @return {WrappedRange}
        */
       createFromNode: function (node) {
-        return this.create(node, 0, node, 1);
+        var sc = node;
+        var so = 0;
+        var ec = node;
+        var eo = dom.nodeLength(ec);
+
+        // browsers can't target a picture or void node
+        if (dom.isVoid(sc) || dom.isImg(sc)) {
```

Comment:
Could you remove `dom.isImg` condition?
`dom.isImg` is unnecessary, because `isVoid` function contains that.

https://github.com/summernote/summernote/blob/develop/src/js/core/dom.js#L98